pkg/machines/config.es6

```diff
@@ -25,7 +25,14 @@
 const VMS_CONFIG = {
     DefaultRefreshInterval: 10000, // in ms
     Virsh: {
-        ConnectionParams: ['-c', 'qemu:///system']
+        connections: {
+            'system': {
```

Comment:
Adding translation to helper.es6:rephraseUI() since these connection-name-strings are visible for the user.